torch_dftd/functions/dftd3.py

```diff
@@ -253,7 +253,7 @@ def edisp(
         if batch.size()[0] == 0:
             n_graphs = 1
         else:
-            n_graphs = int(batch[-1]) + 1
+            n_graphs = cell.size(0)
```

Comment:
This is the only change in this PR, right?

Reply:
I think `cell` can be `None` when the atoms does not have pbc (periodic boundary condition).

Reply:
https://github.com/pfnet-research/torch-dftd/blob/13432143dd350cfbefc7aca5f7c2deed8a45f1fb/torch_dftd/torch_dftd3_calculator.py#L162-L169

Sorry I understand that cell is always filled when batch_calculate is called.

Reply:
Thanks for confirming it. Yes, it can be assumed when the input is not a batch.